### ISSUE TITLE
Difficult to search compute.io packages

### ISSUE THREAD

**Issue by rreusser** [compute-io/contributing#1]
**Problem**: Github's organization search page only searches package _names_, not _descriptions_. Unless there's an index or search method I'm not aware of (google trickery excluded), this makes it difficult to locate the function you need. [compute.io](https://github.com/compute-io/compute.io) has lots of things on one page, but the goal there is a bit different.

**Example**: I accidentally rewrote at least a couple scijs modules because of this.

**Possible solution**: Is it reasonable to suggest a script that would scrape compute.io's (and sibling organizations') repo READMEs, dumping them into an elasticsearch database and exposing it on a simple website? I think it'd be pretty simple to throw something quick up onto, for example, heroku.

**Reason for suggesting what's probably an overengineered solution**: I don't want to get too far ahead of things, but I'm trying to understand how the hundreds-of-single-concern-repos approach (which I think is totally valid) can produce a result that has a chance at general adoption.

**Also**: Is this a good place to ask such questions?

**Comment by kgryte** [comment]
Agreed. A more general search solution is needed. This will hopefully be developed in parallel starting late July. And definitely, throwing the READMEs into ES is on the roadmap. 

Re: module based development. The approach does present organizational issues not encountered in monolithic approaches. Efforts are being made offline to come up with solutions. More to come on this front.

**Comment by rreusser** [comment]
Cool, yeah, I guess, as with everything else, if this is a general/emerging pattern, then there's a more general solution hiding somewhere in here. As long as it's on the radar, I'm content, mainly because manual management of the documentation might get unwieldy if/when more bugfixes, api changes, and contributors start to enter the picture.
